### ISSUE TITLE
In small-scale datasets, why is the training accuracy of models with large models lower than that of models with small models

### ISSUE THREAD

**Issue by lingchensimon** [meituan/YOLOv6#384]
![image](https://user-images.githubusercontent.com/92513876/182137882-b831ede9-a47b-4681-877a-83a329334206.png)

**Comment by mtjhl** [comment]
If the dataset is small, may be large model overfit the trainset.

**Comment by Zephyr69** [comment]
No way to tell with this little info.

Providing both train and val loss trajectories would be a good start.
